source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst

```diff
@@ -1,6 +1,6 @@
 .. _increase-virtual-disk-size:
 
-Increase virtual disk size of a Clear Linux image 
+Increase virtual disk size of a Clear Linux\* image 
```

Comment:
Do you want to replace this with the |CL| substitution?

Reply:
No. This is an exception. For the title of the document, we have to spell out the full name. 
Any other edits?

Reply:
@mvincerx ok, no other edits from me. Thank you!